### ISSUE TITLE
Layout weekly not correct on the iPad

### ISSUE THREAD

**Issue by ortys** [patchthecode/JTAppleCalendar#1334]
**(Required) Version Number:** 8.0.2

## I used the JTAppleCalendar for my iPhone app. I need a weekly view, nothing more.
Now that I'm developing the iPad version of the same app, I'm taking advantage of the same class I have on the iPhone with the same features, just enhancing the layout with a second storyboard.
I simply reproduce what I have on Main.storyboard on the Main_iPad.storyboard version, reading the same classes with the same functions but arranging the objects differently.

If the same weekly collection in the iPhone storyboard works correctly (I attach image), if I set the contraist of its size on the iPad storyboard, it does not behave correctly.
If I change orientation, even worse.
I just want it to show 7 days and for the cells to change their width to fit the width of the superview, like a simple collection.

<img width="1585" alt="Schermata 2021-07-17 alle 12 24 08" src="https://user-images.githubusercontent.com/54210957/126034299-5770f93b-56de-44d5-9af4-7df3d2bd2dde.png">
<img width="1050" alt="Schermata 2021-07-17 alle 12 25 15" src="https://user-images.githubusercontent.com/54210957/126034302-8ab34df7-78c7-43a2-b8b1-a1c33cb21cfd.png">
<img width="636" alt="Schermata 2021-07-17 alle 12 25 50" src="https://user-images.githubusercontent.com/54210957/126034303-47cb33b1-3463-4a38-9e0b-35d5d3575348.png">

**Comment by patchthecode** [comment]
is your calendar fully setup with constraints with zero errors?

**Comment by ortys** [comment]
Yes, no errors. On iPhone works well, on iPad I don't know why there is this problem.
If I don't modify the weight of the collection, and I maintain the same dimension of iPhone collection, it works well.

**Comment by patchthecode** [comment]
hmm.. you have to give me a minimal project example that replicates this issue.
emphasis on minimal.

**Comment by ortys** [comment]
<img width="424" alt="Schermata 2021-08-02 alle 12 15 24" src="https://user-images.githubusercontent.com/54210957/127845989-bdbcff2e-c054-4136-ac03-acc18892cf0c.png">
UPDATE: Now I have some warnings!

**Comment by patchthecode** [comment]
OK.
1. What xcode version are you using?
2. What about the last error, is it still there?

**Comment by ortys** [comment]
1. Xcode Version 12.5.1 (12E507)
2. Yes, I can't correct this issue. There is another problem, when I change the orientation, the collection view update the layout, but the cell inside not

**Comment by ortys** [comment]
![Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2021-08-02 at 14 32 28](https://user-images.githubusercontent.com/54210957/127862474-84e61a3d-df25-4c8e-b4a5-2697a8df61d1.png)
